workers/generic-worker/process/multiuser_windows.go

```diff
@@ -47,6 +43,7 @@ func TaskUserPlatformData() (pd *PlatformData, err error) {
 		return
 	}
 	pd.CommandAccessToken = pd.LoginInfo.AccessToken()
+	win32.DumpTokenInfo(pd.CommandAccessToken)
```

Comment:
where does it dump token info? to the stdout and into logs directly?

Reply:
Yes, exactly!